basic/src/tags/actions.ts

```diff
@@ -0,0 +1,26 @@
+import { Tag } from "wasp/entities";
+import { HttpError } from "wasp/server";
+import { CreateTag } from "wasp/server/operations";
+
+type CreateTagArgs = {
+  name: Tag["name"];
+  color: Tag["color"];
+};
```

Comment:
Maybe something like this:
```suggestion
type CreateTagArgs = Pick<Tag, "name" | "color">
```

I know that this is less open to extending later on.

Reply:
I think i left it as that since I changed it a lot.
Should have tidied it back.